sync-service/src/main/java/com/pavax/stayhome/syncservice/infrastructure/checksum/NodefetchV1CompatibleChecksumStrategy.java

```diff
@@ -0,0 +1,42 @@
+package com.pavax.stayhome.syncservice.infrastructure.checksum;
+
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+import com.pavax.stayhome.syncservice.service.BusinessRequestDto;
+import com.pavax.stayhome.syncservice.service.ChecksumStrategy;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.springframework.util.DigestUtils;
+
+
+@Component
+@Order(100)
+@ConditionalOnProperty(value = "sync-service.checksum.strategy", havingValue = "nodefetch-v1", matchIfMissing = false)
+class NodefetchV1CompatibleChecksumStrategy implements ChecksumStrategy {
```

Comment:
@mageo I might have found a way to make it compatible -> we'd need to start it with the environment variable to try to make the cheksums for the data-provider nodefetch compatible to the version that we have now on master. If the environment variable is not defined then the a new checksum strategy is taken into accound and thus a new one is created 

```
SYNC_SERVICE_CHECKSUM_STRATEGY=nodefetch-v1
```

Reply:
If it is not needed we can omit this env-variable and new checksum will be generated

Reply:
we (@mageo  and I ) decided that we are going to use the new checksum strategy - thus we will keep the complexity lower here in the sync-service and do a propper migration then on the backend/django side